content/en/docs/release-notes/release-notes-0.3.md

```diff
@@ -53,80 +53,80 @@ a flurry of contributions adding support for the variety of providers out there.
 
 With this release, we support the following DNS providers when solving ACME DNS01 challenges:
 
-- Akamai FastDNS (#322, @twz123)
+- Akamai FastDNS (#322, `@twz123`)
 - Amazon Route53
-- Azure DNS (#246, @mwieczorek)
+- Azure DNS (#246, `@mwieczorek`)
 - CloudFlare
 - Google CloudDNS
 There are pull requests in flight to add support for:
-- DNSPod (#486, @hemslo)
-- DNSimple (#483, @marc-sensenich)
-- DigitalOcean (#345, @dl00)
-- INWX (#336, @steigr)
-- RFC2136 (#245, @simonfuhrer)
+- DNSPod (#486, `@hemslo`)
+- DNSimple (#483, `@marc-sensenich`)
+- DigitalOcean (#345, `@dl00`)
+- INWX (#336, `@steigr`)
+- RFC2136 (#245, `@simonfuhrer`)
 
-# Changelog
+# Change log
```

Comment:
I feel we should keep these as "Changelog" and add it to the dictionary. "Changelog" is a technical word